strawberry/types/fields/mutation.py

```diff
@@ -0,0 +1,6 @@
+from strawberry.types import StrawberryField
+
+
+# TODO: What advantage does a separate type provide?
+class StrawberryMutation(StrawberryField):
+    ...
```

Comment:
not much to be honest, for decorators it helps with understand what’s a mutation and what’s a field (when reading code) :)

Reply:
I could see it _maybe_ making type annotation easier in the future